src/researches/passiveVoice/morphological/determinePassiveSentence.js

```diff
@@ -0,0 +1,49 @@
+const filter = require( "lodash/filter" );
+const getWords = require( "../../../stringProcessing/getWords.js" );
+
+// Verb-form lists per language
+const getPassiveVerbsRussian = require( "../../russian/passiveVoice/participlesShortenedList.js" )().all;
+
+/**
+ * Matches the sentence against passive verbs.
+ *
+ * @param {string} sentence The sentence to match against.
+ * @param {Array} passiveVerbs The array containing passive verb-forms.
+ * @returns {Array} The found passive verbs.
+ */
+let matchPassiveVerbs = function( sentence, passiveVerbs ) {
+	return filter( passiveVerbs, function( word ) {
+		let sentenceWords = getWords( sentence );
+		return sentenceWords.includes( word );
+	} );
+};
+
+/**
+ * Checks the passed sentences to see if they contain passive verb-forms.
+ *
+ * @param {Array} sentence The sentences to match against.
+ * @param {Object} language The language of the text.
+ * @returns {Array} Array of sentence objects containing the complete sentence and the transition words.
```

Comment:
This return description doesn't match the code ;)

Reply:
Agreed, fixed.